serdect/tests/messagepack.rs

```diff
@@ -0,0 +1,61 @@
+//! messagepack-specific tests.
+
+#![cfg(feature = "alloc")]
+
+use hex_literal::hex;
+use proptest::{array::*, collection::vec, prelude::*};
+use serdect::{array, slice};
+
+/// Example input to be serialized.
+/// Last byte is `0xFF` to test that no packing is performed for values under 128.
+const EXAMPLE_BYTES: [u8; 16] = hex!("000102030405060708090A0B0C0D0EFF");
+
+/// messagepack serialization of [`EXAMPLE_BYTES`] as a slice.
+const MESSAGEPACK_SLICE: [u8; 18] = hex!("C410000102030405060708090A0B0C0D0EFF");
+
+/// messagepack serialization of [`EXAMPLE_BYTES`] as an array.
+/// Note the 0xCC marker before 0xFF, denoting that the integers are dynamically sized.
```

Comment:
Same as before.

(I think we don't need this anymore, because now bytes and slice have the exact same output.)

Reply:
Fixed